modules/filters.js

```diff
@@ -19,7 +19,7 @@ try {
 
 function getCachebustedPath(urlPath, useRemoteAssets = config.get('features.useRemoteAssets')) {
     const version = assets.version || 'latest';
-    const baseUrl = useRemoteAssets ? 'https://media.biglotteryfund.org.uk/assets' : `/assets`;
```

Comment:
If we're going to change this here we don't need the domain name anymore, or the `useRemoteAssets` flag, this can return `/assets` all the time.

Reply:
Ah I thought we still needed/used this for the email templates. Just removed it!